jsonpickle/pickler.py

```diff
@@ -85,13 +107,17 @@ def __init__(self,
                  keys=False,
                  warn=False,
                  max_iter=None,
-                 numeric_keys=False):
+                 numeric_keys=False,
+                 use_decimal=False,
+                 use_base85=False,
+                 fail_safe=None):
         self.unpicklable = unpicklable
         self.make_refs = make_refs
         self.backend = backend or json
         self.keys = keys
         self.warn = warn
         self.numeric_keys = numeric_keys
+        self.use_base85 = use_base85 and (not PY2)
```

Comment:
The parens here are superfluous.

Reply:
I didn't make that change lol. Github's compare view was wonky. I just updated it and now it has a better view.